### DISCUSSION TITLE
Azure DevOps and Azure Solutions Architects [180-190PLN]

### DISCUSSION THREAD

**Discussion by arczibdg** [AzureCommunityPL/group#368] [Oferty pracy]
Cześć,

Dla klienta z którym współpracuje poszukujemy Azure DevOpsów i Azure Solution Architektów.
Praca długofalowa, jak również możliwe projekty na krótszy okres.

✔Opis stanowiska dla Azure DevOps:
- IaC deployment using ARM, PowerShell and Terraform
- Builiding CICD pipleines in Azure DevOps
- Contributing to centralised code base
- Preparation of Low Level Design documentation
- Ensuring Microsoft and industry security best practices are adhered to
- Aligning deliveries to Microsoft Cloud Adoption Framework (CAF)

✔Wymagania:
- 2 years experience delivering PaaS/IaaS solutions
- Experience working within an Agile team
- Excellent automation skills - PowerShell, ARM and/or Terraform
- Understanding of basic Azure resources
- Experience with configuration Hub-Spoke network topologies, S2S/P2S VPN,  Virtual Network (vNET) peering, Network Security Groups (NSG) etc.

💰Stawka: 170-180 PLN/h + Vat
🎯Lokalizacja: Full Remote

✔Opis stanowiska dla Azure Solutions Architects:
- Architect, design and deploy Azure Landing Zones using ARM, PowerShell and Terraform
- Following CICD best practices in Azure DevOps
- Leading Agile cloud transformation projects
- Contributing to centralised code base
- Preparation of HLD and LLD documentation
- Advise and guide clients on technical solutions which meet their requirement
- Deep understanding of Microsoft Cloud Adoption Framework and ability to align these principles to cloud delivery projects.
- Leading client facing Knowledge Transfer (KT) sessions

✔Wymagania:
- 3 years experience delivering and designing Azure IaaS and PaaS solutions
- Experience implementing highly available infrastructure.
- Excellent automation skills - PowerShell, ARM and/or Terraform
- Understanding of basic Azure resources
- Experience with configuration Hub-Spoke network topologies, S2S/P2S VPN,  Virtual Network (vNET) peering, Network Security Groups (NSG) etc.
- Experience deploying Highly Available Virtual Machines using Availability Sets & Availability Zones.
- Demonstrable risk management knowledge

💰Stawka: 180-190 PLN/h + Vat
🎯Lokalizacja: Full Remote

Jeśli chcecie zaaplikować albo macie jakieś pytania proszę o kontakt na hr@incredevo.com.

Pozdrawiam

**Comment by smereczynski** [comment]
@wi5nia czy my nie wymagamy podawania nazwy klienta?

**Reply by eklime** [reply]
No co najmniej lokalizacji firmy, nawet przy Full Remote.
No i wpłaty na konto, gdy znajdzie się kandydat przez naszą grupę 🤔

**Reply by wi5nia** [reply]
@smereczynski wymagamy... 

@arczibdg - popraw post albo bedzie usuniety.
